src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.dao.impl.BookDaoImpl;
+import mate.academy.model.Book;
+
 public class Main {
     public static void main(String[] args) {
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("mate.academy");
    
    public static void main(String[] args) {
```

Reply:
what the injector is? why do you need it at all?

Reply:
See the task description

Reply:
Please either correct the condition or explain in the condition what is the condition for using the injector. All that is said in the condition 'it MAY look like that'
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/71705897/f7898d23-4510-4901-8b97-c250b9056562)

Also there is nothing about injector in 'common mistakes'

Reply:
Let's discuss your questions in chat